src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@sebing In this case the problem is in `RqLive`, let's create a unit test for it which reproduces the bug, ignore it, and tell the solver of the puzzle to fix the bug and to remove `@Ignore` from that test as well as this one.